components/event-sources/Makefile

```diff
@@ -180,6 +187,16 @@ injection: bin/injection-gen
 		--versioned-clientset-package $(PKG)/client/generated/clientset/internalclientset \
 		--external-versions-informers-package $(PKG)/client/generated/informer/externalversions
 
+istio-injection: bin/injection-gen
+	@echo "+ Generating injection for $(ISTIO_API_GROUPS)"
+	@rm -rf client/generated/injection/istio
+	@bin/injection-gen \
+        --go-header-file hack/boilerplate.go.txt \
+        --input-dirs $(subst $(space),$(comma),$(istio-api-import-paths)) \
+        --output-package $(PKG)/client/generated/injection/istio \
+        --versioned-clientset-package $(ISTIO_CLIENT_GO_PKG)/clientset/versioned \
```

Comment:
Any reason for using a different path that the other generated injection packages?

Reply:
Other generated injection packages need a restructure. It was assumed that injection clients can only have event-sources. But this is not true anymore.
My ideal world will be:
- `event-sources/client/generated/injection/`
  - istio (but this is the first step to the right direction)
  - sources (which I don't prefer to do it in this PR)

Reply:
Ohh true, my bad, please ignore my comment!